src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/AESetup.java

```diff
@@ -0,0 +1,158 @@
+                + cekName + ") NULL" + ");";
+
+        try {
+            stmt.execute(sql);
+        }
+        catch (SQLException e) {
+            fail(e.toString());
+        }
+    }
+
+    private static void createCMK(String keyStoreName,
+            String keyPath) throws SQLException {
+        String sql = " if not exists (SELECT name from sys.column_master_keys where name='" + cmkName + "')" + " begin" + " CREATE COLUMN MASTER KEY "
+                + cmkName + " WITH (KEY_STORE_PROVIDER_NAME = '" + keyStoreName + "', KEY_PATH = '" + keyPath + "')" + " end";
+        stmt.execute(sql);
+    }
+
+    static void createCEK(SQLServerColumnEncryptionKeyStoreProvider storeProvider,
+            String certStore) throws SQLException {
+        String letters = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
+        byte[] valuesDefault = letters.getBytes();
+        String cekSql = null;
+        if (certStore.equalsIgnoreCase("MSSQL_JAVA_KEYSTORE")) {
+            byte[] key = storeProvider.encryptColumnEncryptionKey(javaKeyAliases, "RSA_OAEP", valuesDefault);
+            cekSql = "CREATE COLUMN ENCRYPTION KEY " + cekName + " WITH VALUES " + "(COLUMN_MASTER_KEY = " + cmkName
+                    + ", ALGORITHM = 'RSA_OAEP', ENCRYPTED_VALUE = 0x" + DatatypeConverter.printHexBinary(key) + ")" + ";";
+        }
+        else if (certStore.equalsIgnoreCase("MSSQL_CERTIFICATE_STORE")) {
+            String encryptedValue = "0x016E000001630075007200720065006E00740075007300650072002F006D0079002F0066006200640066003900360031003600360031003100390066006600390039006200380032003800300064003200390064003100360030006600610065006300370030006300640031003100620034002DC298A90D6C4FB6EE59BD1F4E58E3CE334B33E4786608B0A29B8B6FDD376F9C42716E00077D91FE80659EB427F1D5509971D24B3B7CB761E79CBD894CBE8EE0009DE4DB9ABECCC398F80AD8B95E3A89692E91BCF6B0518552CFD224816F67E0C37D48B538E38A91A9BA73D6CF84F315560BCB69423D0F4682FDE1DD12412823362641E6B7F19843390D2BE9E26BDA0FCAB01F987EF7AA882468EE86FAB6FE29C771FB22BEF355377B158DA06D9998171110A21AEEDA875851CE8BC64A49D00925AD844F47150F27B6147DAACE1E4B93C9E2B9B91BF5B26BD6FE10EF0C2EDC9395A9E5D2B007E6F16229ABC27068C07F7A77EC32F24FCFE04D53CF260A58440009F8B70E4A9091426159189C021A25D52E7FEA9B341DAC5361C41F3E32800D31A10EF193E4F58DE161302C1E0607B1FA56288FA4592F3F269173D4177BB77EEFCA6B99052EE9A8725B121A731981133C25414634DAB47040A7AED2EAFBA459FF1CA6A19C500A305C2154D9E64B4DD79D8B7394703756A4BCE39782BC5C3E6C9FAC088149554F5AED125FBFC081CFEE8FA83153135BE10718167AF4114F37CA10925A690D94BF53C69AF4BE6F8CAE74450BCDE312E2074D9F5788E57C515A507B86E64B54AC3624F3F8A29C9007C798518304766F6862A0824108143B2E532B82442816A9D89A9585E343CEE6480F7AC881584CA14F5A929A7FF3562D57B40305";
+            cekSql = "CREATE COLUMN ENCRYPTION KEY " + cekName + " WITH VALUES " + "(COLUMN_MASTER_KEY = " + cmkName
+                    + ", ALGORITHM = 'RSA_OAEP', ENCRYPTED_VALUE = " + encryptedValue + ")" + ";";
+
+        }
+        connection.createStatement().execute(cekSql);
```

Comment:
Should we use connection from parent class or connection from current class ? If any one is Ok then better stick with current connection. i.e. `con`